.eslintrc

```diff
@@ -28,5 +28,6 @@
     "react/jsx-filename-extension": [1, { "extensions": [".js"] }],
     "react/jsx-props-no-spreading": 0,
     "react/no-array-index-key": 1,
+    //"linebreak-style": ["error", "windows"],
```

Comment:
Do we add it again?

Reply:
comment it out, so I can use during my local